src/data/fintype/basic.lean

```diff
@@ -404,11 +404,11 @@ by { rw ← subtype_card s H, congr }
 def of_finset {p : set α} (s : finset α) (H : ∀ x, x ∈ s ↔ x ∈ p) : fintype p :=
 fintype.subtype s H
 
-@[simp] theorem card_of_finset {p : set α} (s : finset α) (H : ∀ x, x ∈ s ↔ x ∈ p) :
+theorem card_of_finset {p : set α} (s : finset α) (H : ∀ x, x ∈ s ↔ x ∈ p) :
   @fintype.card p (of_finset s H) = s.card :=
 fintype.subtype_card s H
 
-theorem card_of_finset' {p : set α} (s : finset α)
+@[simp] theorem card_of_finset' {p : set α} (s : finset α)
```

Comment:
Both of these changes are incorrect, I think: the first one triggers only on `fintype.card` applied to a specific instance `of_finset` and so is a reasonable simp lemma, while the second one is only a conditional rewrite since nothing fixes `H`.

Reply:
Oh, thanks, not sure what I was thinking. The second one doesn't even fix `s`.